lib/request/GetRequestV2.js

```diff
@@ -104,14 +104,21 @@ GetRequestV2.prototype = {
                             var currentVersion = incrementVersion.getCurrentVersion();
                             currentCacheVersion.setVersion(currentVersion);
                             var mergeContext = {hasInvalidatedResult : false};
-                            self._merge(rPaths, err, data, mergeContext);
+
+                            var pathsErr = model._useServerPaths && data.paths === undefined ?
+                                new Error("Server responses must include a 'paths' field when Model._useServerPaths === true") : undefined;
```

Comment:
Is data guaranteed to be an object when err is an error?

Reply:
Will add a null check just in case.